tests/functional/sync_sign_in.js

```diff
@@ -109,8 +109,7 @@ define([
         .get(require.toUrl(PAGE_URL_WITH_MIGRATION))
         .setFindTimeout(intern.config.pageLoadTimeout)
         .execute(listenForFxaCommands)
-        .findByCssSelector('.info.nudge')
-        .isDisplayed()
```

Comment:
`.info.nudge` shows up in the dom faster than it is displayed, need to use `visibleByQSA`